tests/test_core.py

```diff
@@ -328,7 +329,20 @@ def test_stack_map_offset(s_dict, t_dict, f_in, result):
     s = AnatomicalSpace(**s_dict)
     t = AnatomicalSpace(**t_dict)
 
-    assert np.allclose(t.map_stack_to(s, np.ones((2, 2, 2)), **f_in), result)
+    if np.any(np.array(t_dict["offset"]) < 0):
+        ctx = pytest.warns(
+            UserWarning,
+            match=(
```

Comment:
@vigji is the warning that's caught here correct?

Reply:
Yes it is!